src/wmnds/components/content-tiles/travel-updates/_travel-updates.scss

```diff
@@ -1,3 +1,44 @@
-.wmnds-content-tile-container {
-  width: fit-content;
+.wmnds-content-tile--container {
+  &__wrapper {
+    width: 335px;
+    margin-bottom: $size-sm;
+  }
+
+  &__star {
+    display: block;
+    width: 2rem;
+    height: 2rem;
+    margin-left: 3.5rem;
+    fill: get-color("secondary");
+  }
+
+  &__bus {
+    margin-right: $size-lg;
+    margin-bottom: $size-sm;
+  }
+
+  .wmnds-disruption-indicator-medium--narrow {
```

Comment:
We shouldn't be overwriting other component styles as a component should look the same everywhere.
As this class will also overwrite the component used anywhere else.

If you need to modify it, give it a brand new class related to the component/pattern you're styling...in this case it is travel-updates.